qa/tasks/ceph_test_case.py

```diff
@@ -140,12 +140,14 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
         return ContextManager()
 
-    def wait_for_health(self, pattern, timeout):
+    def wait_for_health(self, pattern, timeout, check_in_detail=None):
         """
         Wait until 'ceph health' contains messages matching the pattern
+        Also check if @check_in_details is matches detailed health messages
```

Comment:
nit:
s/@check_in_details/@check_in_detail ?
s/is matches/matches ?

Reply:
Sure.